VideoWeb/VideoWeb/ClientApp/src/app/shared/directives/tooltip.directive.ts

```diff
@@ -62,6 +67,9 @@ export class TooltipDirective implements OnDestroy {
         this.create();
         this.updatePosition($event);
         this.show();
+        setTimeout(() => {
```

Comment:
this looks like it will hide is on desktop and mobile after a timeout. surely you want the tooltip to stay open whilst there's a pointing device on it like when there's a mouse on a desktop. Would you mind checking on this please?

Reply:
it won't hide tooltip on desktop becuase the mouseenter and mouseleave event will controll the tooltip completely, the timeout do nothing in desktop and we expect to hide the tooltip in mobile after  a timeout